src/components/select/examples/select-initially-empty.tsx

```diff
@@ -1,8 +1,14 @@
-import { Option } from '@limetech/lime-elements';
+import { LimelSelectCustomEvent, Option } from '@limetech/lime-elements';
 import { Component, h, State } from '@stencil/core';
 
 /**
- * Initially Empty
+ * With Empty Option
+ *
+ * Adding an empty option makes it possible for the user to "unset"
+ * the value. Try selecting a value below, and then selecting the empty
+ * option again.
+ *
+ * If the component is set as required, the empty option is removed.
```

Comment:
I updated this example, because it's not necessary to have an empty option to make the component initially empty. I think that was necessary when we first built the `limel-select` component, and that's why this example was here. But it doesn't really make sense anymore, since the first example, which doesn't have an empty value, is also initially empty.

So I changed this to make more sense and show off the functionality we just added in this PR 🙂